node-manager/src/chain.rs

```diff
@@ -343,7 +345,10 @@ impl MutinyChain {
             }
         }
 
-        *self.watched_transactions.lock().unwrap() = to_watch;
+        let mut locked_watched_transactions = self.watched_transactions.lock().unwrap();
+        for txid in to_watch {
+            locked_watched_transactions.insert(txid);
+        }
```

Comment:
Should we `.clear()` first or do we really need to just be inserting new ones to add to it? The old behavior did a replacement.

Reply:
I think we are supposed to add, that's what is done in the ldk PR